src/WireMock.Net/Owin/OwinSelfHost.cs

```diff
@@ -26,7 +27,7 @@ public OwinSelfHost([NotNull] WireMockMiddlewareOptions options, [NotNull] param
 
             _logger = options.Logger ?? new WireMockConsoleLogger();
 
-            foreach (string uriPrefix in uriPrefixes)
+            foreach (string uriPrefix in uriPrefixes) // .Select(u => u.EndsWith("/") ? u : $"{u}/")) // Always append an / at the end.
```

Comment:
Is the code supposed to be commented out?

Reply:
Yes should be removed.